internal/log/scrub.go

```diff
@@ -135,7 +153,13 @@ func scrubBytes(b []byte, scrub scrubberFunc) ([]byte, error) {
 	if err := encode(buf, m); err != nil {
 		return nil, err
 	}
-	return buf.Bytes(), nil
+
+	// remove newline encode appends
+	bb := buf.Bytes()
+	if len(bb) > 0 && bb[len(bb)-1] == '\n' {
+		bb = bb[:len(bb)-1]
+	}
```

Comment:
Shouldn't this be using the `trimStringNewline` or the golang strings's call to do the same?